### PR TITLE
clientconfig: Add support for SSL setting to disable verification

### PR DESCRIPTION
**Pull request by haxorof**
For #45

@jtopjian : Please review when you got time. I was a bit unsure here what shall be done here. For example in Terraform Cloud struct is fetched [here](https://github.com/terraform-providers/terraform-provider-openstack/blob/master/openstack/config.go#L76) then it is easy to just fetch the values directly. Or shall some function be introduced?

### CODE REVIEW

File: openstack/clientconfig/results.go [hunk 1]
```diff
@@ -17,9 +17,25 @@ type Cloud struct {
 	// API Version overrides.
 	IdentityAPIVersion string `yaml:"identity_api_version"`
 	VolumeAPIVersion   string `yaml:"volume_api_version"`
+
+	// Verify whether or not SSL API requests should be verified.
+	// Note! Type is not set to bool to be able to set correct default value.
+	Verify interface{} `yaml:"verify"`
```
Comment (jtopjian) [inline openstack/clientconfig/results.go]: We can change this to `*bool` and the YAML package will account for it accordingly.

This way we can check for nil, true, and false.

File: openstack/clientconfig/testing/fixtures.go [hunk 2]
```diff
@@ -14,6 +14,7 @@ var HawaiiCloudYAML = clientconfig.Cloud{
 		ProjectName: "Some Project",
 		DomainName:  "default",
 	},
+	Verify: true,
```
Comment (jtopjian) [inline openstack/clientconfig/testing/fixtures.go]: At the top of this file, do:

```go
var iTrue bool = true
var iFalse bool = false
```

And then set:

```go
Verify: &iTrue
Verify: &iFalse
```

where appropriate.

**Comment by haxorof** [comment]
Thanks @jtopjian! I will fix this straight away. Should have thought of `*bool` but must have been sleeping 😄

**Comment by haxorof** [comment]
Now it is hopefully corrected.

**Comment by jtopjian** [comment]
@haxorof Looks good to me. Thank you for implementing this :)

**Pull request merged**